scheduler/src/cook/mesos/api.clj

```diff
@@ -1093,7 +1093,8 @@
   {:job [s/Uuid]})
 
 (def UpdateRetriesRequest
-  {(s/optional-key :jobs) [s/Uuid]
+  {(s/optional-key :job) s/Uuid
```

Comment:
Ah okay.

I think we should allow only one or the other then and report an error if both fields are specified.